formulas/functions/statistical.js

```diff
@@ -215,7 +215,23 @@ const StatisticalFunctions = {
 
   MAX: () => {},
 
-  MAXA: () => {},
+  MAXA: (...values) => {
+    const arr = [];
+    arr.push(...values.map(a => a.value))
```

Comment:
`Array.map` returns a new array already. Spreading it (`...values`) just to `push()` the result is unneccessary. 

```suggestion
    const arr = values.map(a => a.value);
```